tox.ini

```diff
@@ -3,13 +3,13 @@ envlist = py36
 
```

Comment:
Getting the following running `pipenv run tox`
```
Traceback (most recent call last):
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/bin/tox", line 11, in <module>
    sys.exit(cmdline())
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/session.py", line 39, in main
    config = prepare(args)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/session.py", line 27, in prepare
    config = parseconfig(args)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/config.py", line 245, in parseconfig
    parseini(config, inipath)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/config.py", line 810, in __init__
    config.envconfigs[name] = self.make_envconfig(name, section, reader._subs, config)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/config.py", line 838, in make_envconfig
    res = meth(env_attr.name, env_attr.default, replace=replace)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/config.py", line 1011, in getargvlist
    return _ArgvlistReader.getargvlist(self, s, replace=replace)
  File "/Users/benjefferies/.virtualenvs/ras-collection-instrument-QrNRiIIx/lib/python3.6/site-packages/tox/config.py", line 1199, in getargvlist
    (reader.section_name, "commands"))
tox.ConfigError: ConfigError: line-continuation ends nowhere while resolving for [testenv:py36] commands
```

Reply:
Snipped off the [] for some reason.